models/volumeSnapshotDataSource.go

```diff
@@ -0,0 +1,34 @@
+package models
+
+import "github.com/hashicorp/terraform-plugin-framework/types"
+
+// VolumeSnapshotDataSource - Volume Snapshot DataSource properties
+type VolumeSnapshotDataSource struct {
```

Comment:
Is it different from volume model? if not, can we re-use the same?

Reply:
updated